sdk/appconfiguration/azure-appconfiguration-provider/dev_requirements.txt

```diff
@@ -4,3 +4,4 @@
 -e ../../keyvault/azure-keyvault-secrets
 aiohttp>=3.0
```

Comment:
aiohttp might also need third party license notice

Reply:
This library isn't packaged with us, so I wouldn't expect so. These are dev requirements that are used in development.